conf/application.conf

```diff
@@ -235,7 +235,7 @@ manage-details{
 }
 
 assets {
-  version = "3.11.0"
+  version = "3.25.0"
```

Comment:
⛏️ This also doesn't seem to have anything to do with color contrast.

Reply:
This changes the library to a version which includes the colour changes which were made in the move from GOVUK Elements to GOVUK Frontend.

Reply:
I would be nitpicking if this was included in the commit message :)